build.gradle

```diff
@@ -380,6 +380,18 @@ project('serializers') {
         testCompile group: 'io.pravega', name: 'pravega-test-testcommon', version: pravegaVersion
     }
     
+    shadowJar {
```

Comment:
why do we need this for json compatibility checker?

Reply:
I do not understand why would this change for the json compatibility checker, which is just a package being added to the project. I think the structure of the build.gradle file remains the same, with shading of dependencies and creation of fat jars.

Reply:
my question is why is this being added ? you are not creating a new module. and all existing projects that we wanted shaded, are already done so.